meson.build

```diff
@@ -0,0 +1,33 @@
+project(
+  'bees',
+  [ 'c', 'cpp', ],
+  default_options : [
+    'warning_level=3',
+    'c_std=c99',
+    'cpp_std=c++11',
+  ],
+  license: 'GPL-3.0-or-later',
+  version : '0.7',
+)
+
+# Helper scripts for source generation are written in python.
+# Use the same python as meson was run with.
+pymod = import('python')
+python = pymod.find_installation()
```

Comment:
Just use `find_program('python3')` and meson will perform the same detection logic.

Using the python module incurs an extra import and runs an introspection command to scrape sysconfig info out of python. The entire purpose of the module is to provide a sysconfig-aware interface for compiling and/or installing python software into site-packages/.